containers/src/Kokkos_Bitset.hpp

```diff
@@ -92,9 +92,10 @@ class Bitset {
   using block_view_type = View<unsigned*, Device, MemoryTraits<RandomAccess>>;
 
  public:
-  /// constructor
+  Bitset() = default;
```

Comment:
Just wondering... Is `KOKKOS_DEFAULTED_FUNCTION` still needed to make the bitset default constructible on device?

Reply:
It was not constructible from device before and I see no good reason to change it here

Reply:
Ok!